src/main/java/com/diffmin/App.java

```diff
@@ -218,13 +225,14 @@ public static void main(String[] args) {
             return;
         }
         try {
-            App app = new App(args[0]);
-            List<Operation> operations = app.getOperations(new File(args[0]), new File(args[1]));
-            app.generatePatch(operations);
+            App app = new App();
+            Pair<Diff, CtModel> diffAndModel =
+                    App.computeDiff(new File(args[0]), new File(args[1]));
+            app.generatePatch(diffAndModel.getFirst());
             app.applyPatch();
-            CtModel patchedCtModel = app.modelToBeModified;
+            CtModel patchedCtModel = diffAndModel.getSecond();
             System.out.println(app.displayModifiedModel(patchedCtModel));
-        } catch (Exception e) {
+        } catch (java.lang.Exception e) {
```

Comment:
This comment completely passed me by. It's sometimes rather tricky to figure out _what_ a GitHub notification refers to...

And yes, this happened because I used Spork, which internally uses Spoon. And there are a ton of corner cases in Spoon when resolving the package of a type, and sometimes it fails to mark it as implicit (and thus, it's printed, like it was here).

Anyway, you can find a lot of test files in the Spork repository, under `src/test/resources/clean/both_modified`. See the README for a quickstart: https://github.com/KTH/spork